packages/style-engine/class-wp-style-engine.php

```diff
@@ -513,3 +566,27 @@ function wp_style_engine_generate( $block_styles, $options = array() ) {
 	}
 	return null;
 }
+
+// @TODO Just testing!
+/**
+ * Global public interface to register block support styles support to be output in the frontend.
+ *
+ * @access public
+ *
+ * @param array $block_styles An array of styles from a block's attributes.
+ * @param array $options An array of options to determine the output.
+ * @return void
+ */
+function wp_style_engine_enqueue_styles( $block_styles, $options = array() ) {
```

Comment:
Here we're generating and enqueuing block styles, a.k.a values from a styles object `{ spacing: { ... }, typography: { ... }, ... }` 

So maybe it should be called `wp_style_engine_enqueue_block_styles`.